MOxUnit/moxunit_runtests.m

```diff
@@ -142,3 +156,28 @@
         filenames={pwd()};
     end
 
+
+function write_xml(fname, test_result)
+
+    if isempty(fname);
+        % No file to write to, so stop now.
+        return;
+    end
+
+    % Open file to start writing XML report
+    fid=fopen(fname,'w');
+    if fid==-1
+        error('Could not open file %s for writing', fn);
+    end
+
+    % Start with the XML header
+    fprintf(fid, '<?xml version="1.0" encoding="utf-8"?>');
```

Comment:
When using the TestResult class, the .stream property could be used directly in fprintf there - and output can be written to standard output (fid=1) or to a file.